src/commands/install.yml

```diff
@@ -21,20 +24,26 @@ parameters:
     default: amd64
 steps:
   - run:
-      name: Get download URL
+      name: Warn about deprecations
+      command: |
+        if [ -n "<< parameters.channel >>" ]; then
+          echo "The 'channel' parameter is deprecated and no longer has any effect!"
+          echo "See https://github.com/rainforestapp/rainforest-orb/releases for more info."
```

Comment:
Should we give them the quick fix as part of the deprecation message; in case it takes some time/we've done a bunch of releases to the orb before they get around to this?

i.e. "The channel parameter is deprecated and no longer has any effect! Please update your configuration to use the version parameter instead. See <url> for more info.".

Reply:
It doesn't look like there's a way to both raise the deprecation to the end user's attention and not fail the build. I added `when: always` to the install step after the deprecation step, so it'll always install the CLI even if the deprecation warning was raised, but then it'll still fail after that (see https://app.circleci.com/pipelines/github/rainforestapp/rainforest-orb/276/workflows/5bd40a02-70d6-4cab-861b-ff58396379b3/jobs/614)

Given that, I'll remove the `when: always` and have it fail loudly (which is what would happen if we didn't have the check to begin with).